Packs/GigamonThreatINSIGHT/Integrations/GigamonThreatINSIGHT/GigamonThreatINSIGHT.py

```diff
@@ -9,61 +9,313 @@
 from CommonServerPython import *
 from CommonServerUserPython import *
 
-import requests
 import json
 import re
 from datetime import datetime, timedelta
 
-# Global for setting up headers
-HEADERS: Dict[str, str] = {}
 TRAINING_ACC = 'f6f6f836-8bcd-4f5d-bd61-68d303c4f634'
 MAX_DETECTIONS = 100
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 
 
-''' HELPER FUNCTIONS '''
+class Client(BaseClient):
+    @staticmethod
+    def getUrl(api) -> str:
```

Comment:
This comment isn't specific to your code changes, but I wonder if there is any style guide we are supposed to be following?  PEP 8?
Nothing to do with your changes but I see a mix of styles.  If we know what style we should adhere to we can maybe start moving in that direction.